client/src/components/pages/not-found/not-found.module.css

```diff
@@ -1,11 +1,28 @@
 .not-found {
   width: 100%;
-  height: 223px;
+  min-height: calc(100vh - 200px); /* Account for header and footer space */
   flex-shrink: 0;
   background: var(--color-dark);
+  background-image: url("@/assets/images/beer-tap.png");
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
   box-shadow: 0px 25px 20px 0px rgba(0, 0, 0, 0.45);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
 
   .not-found-content {
     padding: 2rem;
+    text-align: center;
+    background: rgba(
+      46,
+      46,
+      46,
+      0.8
+    ); /* Semi-transparent overlay for readability */
```

Comment:
We have been centralizing color definitions [here](https://github.com/codeforboston/boston-liquor-license-tracker/blob/main/client/src/styles/index.css)

This makes it easy to ensure a consistent style, and to facilitate future changes.

There are quite a few ways to achieve this. I'll leave it for you to choose one, but aim to keep a single source of truth for that color.